.storybook/config.js

```diff
@@ -13,6 +13,7 @@ const req = require.context('../modules', true, /stories.*\.tsx?$/);
 
 function loadStories() {
   req.keys().forEach(req);
+  document.body.dir = 'rtl';
```

Comment:
This sets `dir='rtl'` to the body element, triggering the browser's algorithm to flip most elements to `RTL`. We should use a context to do that at some point.

Reply:
There is a storybook addon: https://www.npmjs.com/package/storybook-addon-rtl

But from talking to the ChromaticQA team, they suggest stories that enable a RTL mode so that we can get RTL snapshots as regular stories rather than trying to globally flip and take screenshots for stories in LTR mode and RTL mode. They said other Design Systems of our scale have done the same thing.

Reply:
@NicholasBoll Our plan was to also add optional rtl props to everything so we could test individually with those couldn't we?

Reply:
Adding `dir="rtl"` should work on any level of the DOM. The story could add this attribute to the story container. The alternative is a decorator that adds the `document.body.dir` change. And a prop for `RTL` for components that need it. Will all components need a RTL prop?